packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -1297,6 +1297,9 @@ class TestWidgetInspectorService extends Object with WidgetInspectorService {
         expect(service.toObject(childJson['valueId']), equals(children[i].value));
         expect(service.toObject(childJson['objectId']), isInstanceOf<DiagnosticsNode>());
         final List<Object> propertiesJson = childJson['properties'];
+        for (Map<String, Object> propertyJson in propertiesJson) {
+          expect(propertyJson.containsKey('children'), isFalse);
```

Comment:
nit: use the `contains` matcher instead of `isFalse` as that should give better error output.

Reply:
(and `not` matcher)